package.json

```diff
@@ -76,7 +76,7 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "dependencies": {
-    "@elastic/eui": "v0.0.47",
+    "@elastic/eui": "v0.0.51",
```

Comment:
Since this is merging into a feature branch, can we keep it bumped until before the feature branch gets merged? There are some major updates between these two versions including a new component that should be used.